net/miniserve/Portfile

```diff
@@ -14,14 +15,28 @@ maintainers         {outlook.de:judaew @judaew} openmaintainer
 
 description         \
     miniserve is a small, self-contained cross-platform CLI tool that allows \
-    you to just grab the binary and serve some file(s) via HTTP. 
+    you to just grab the binary and serve some file(s) via HTTP.
 long_description    \
     ${description} Sometimes this is just a more practical and quick way than doing things properly.
 
 checksums           ${distname}${extract.suffix} \
-                    rmd160  19131bd810209a88698be8ee66f5c1441a600743 \
-                    sha256  4ae787fbfd68a1045251a129c75a9e4608e080fd26d48c96fc79fc9e05a49cbf \
-                    size    128862 
+                    rmd160  6ba8023bdc06bc2e8ceea1ac615478558e629ebe \
+                    sha256  4b961a45b8ef645f4f25c5e860d7464bdb5712a7a49275a6d815f4c1194bbf60 \
+                    size    138194
+
+# Remove the specific reference to download the mime_guess package via Git.
+# See https://github.com/macports/macports-ports/blob/d1ac384e5be77cf36af716173d3dafc351e968cb/net/bandwhich/Portfile#L31-L35
+#
+# This fix and upload via cargo.crates_github will be out of date when there
+# is a new version of mime_guess in the release.
+# https://github.com/abonander/mime_guess/pull/66
+# https://github.com/abonander/mime_guess/issues/68
+patchfiles          patch-Cargo-toml-remove-git-package.diff
```

Comment:
it looks like you forgot to do `git add` for the patchfile?

Reply:
exactly, thanks so much